### PR TITLE
delete coments

### PR DESCRIPTION
**Pull request by KostSvitlana**

### CODE REVIEW

File: backend/views/views.py [hunk 1]
```diff
@@ -63,17 +61,17 @@ def user_page():
                 if len(one_string) < MIN_SEARCH_STR:
                     continue
                 search_parameter = '%{}%'.format(one_string)
-                name_search = User.name.like(search_parameter)
-                alias_search = User.alias.like(search_parameter)
-                email_search = User.email.like(search_parameter)
+                search_like[0] = User.name.like(search_parameter)
```
Comment (NewIvanIvanov) [inline backend/views/views.py:65]: define variable search_like

Comment (KostSvitlana) [reply backend/views/views.py:65]: fixed

Comment (NewIvanIvanov) [reply backend/views/views.py:65]: failed with: IndexError: list assignment index out of range
Please run code on your local machine first

Comment (KostSvitlana) [reply backend/views/views.py]: fexed

**Pull request closed without merging**